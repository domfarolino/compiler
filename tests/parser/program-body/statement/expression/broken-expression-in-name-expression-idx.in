program dominic is

  integer dom;
  integer farolino [0: 10];

begin

  farolino[0] := (farolino[4 +]);

end program.
