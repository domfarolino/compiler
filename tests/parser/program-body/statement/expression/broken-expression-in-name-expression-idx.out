[1;31mLine 8: Error parsing expression operator. Expected valid syntax, but got: ']'
Line 8: Error parsing program body
[0m